pkg/hostname/analyzer.go

```diff
@@ -83,8 +83,8 @@ func (a *analyzer) start() {
 				output.query = bson.M{
 					"$push": bson.M{
 						"dat": bson.M{
-							"ips":     data.ResolvedIPs,
-							"src_ips": data.ClientIPs,
+							"ips":     data.ResolvedIPs.Items(),
```

Comment:
Note that .Items() must be called to convert the map into an array. 

https://github.com/activecm/rita/commit/cd8dd38af38d30b06b0141d08b3370435ef2ba4e
https://github.com/activecm/rita/commit/180de847eb6ddb15e008891d9d690427ef307b45